angular.json

```diff
@@ -815,6 +815,28 @@
                 }
             }
         },
+        "styles": {
+            "root": "projects/styles",
+            "sourceRoot": "projects/styles",
+            "projectType": "library",
+            "prefix": "tui",
+            "architect": {
+                "build": {
+                    "builder": "@angular-devkit/build-ng-packagr:build",
+                    "outputs": ["dist/styles"],
+                    "options": {
+                        "tsConfig": "tsconfig.build.json",
```

Comment:
Do we really need build process for this package ?

Or we can just copy files to `dist/`-directory ?